src/macros-by-example.adoc

```diff
@@ -764,50 +835,63 @@ the type tags are unnecessary and may add up to a non-trivial amount of wasted s
 when you observe that the overhead for each value also includes length information: encoding an
 octet of data takes two octets on the stream.
 
-Ion 1.1 tries to mitigate this overhead in the binary format by allowing macro parameters to have
-tagless types.  These are subtypes of the concrete types,
+Ion 1.1 tries to mitigate this overhead in the binary format by allowing macro parameters to use
+more-constrained _primitive types_.  These are subtypes of the concrete types,
 constrained such that type tags are not necessary in the binary form.  In general this can shave
 4-6 bits off each value, which can add up in aggregate.  In the extreme, that octet of data can
 be encoded with no overhead at all.
 
+The following primitive types are available:
+
+[cols="1,1"]
+|===
+|*Primitive Type*                               |*Description*
+| `*var_symbol*`                                | Tagless symbol (SID or text)
+| `*var_string*`                                | Tagless string
+| `*var_int*`                                   | Tagless, variable-width signed int
+| `*uint*`                                      | Tagless, variable-width unsigned int
```

Comment:
For symmetry with the signed int primitives, I think I'd spell `uint` as `var_uint`.

Reply:
Agree, with the caveat that we gotta re-spell `var_`.

Reply:
`elastic_int` 😛